doc/host_create.md

```diff
@@ -195,26 +194,29 @@ Available keys for `--compute-attributes`:
 ```
 cpus          # number of CPUs
 memory        # string, amount of memory, value in bytes
-start         # Must be a 1 or 0, whether to start the machine or not
```

Comment:
why did you remove start from here? it should still be here in when creating a host.